tests/integration_test_snippet_config.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from syncano.exceptions import SyncanoValueError
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class SnippetConfigTest(InstanceMixin, IntegrationTest):
```

Comment:
Add a test here which check something like this:

1. Create config with `some field`
2. Update the config with `new field`
3. Check if `some field` is still present in config after update.